src/main/scala/cromiam/cromwell/CromwellClient.scala

```diff
@@ -0,0 +1,57 @@
+package cromiam.cromwell
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.stream.ActorMaterializer
+import cromiam.cromwell.CromwellClient._
+import cromwell.api.model.CromwellStatus
+import com.softwaremill.sttp._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import cromiam.server.status.StatusCheckedSubsystem
+import cromwell.api.model.CromwellStatusJsonSupport._
+import spray.json._
+
+import scala.concurrent.{ExecutionContextExecutor, Future}
+
+/**
```

Comment:
most of this class is c/p from CromiamApiService

Reply:
could use a whack with the import optimizer to straighten stuff out

Reply:
it hath been whacked